docs/usage-as-end-user.md

```diff
@@ -84,6 +84,22 @@ The `cloudControllerManager.featureGates` contains a map of explicitly enabled o
 For production usage it's not recommend to use this field at all as you can enable alpha features or disable beta/stable features, potentially impacting the cluster stability.
 If you don't want to configure anything for the `cloudControllerManager` simply omit the key in the YAML specification.
 
+## `WorkerConfig`
+
+Each worker pool in a shoot may contain provider specific configurations and options. These are contained in the `providerConfig` section of a worker pool.
+An example of a `WorkerConfig` looks as follows:
+
+```yaml
+apiVersion: openstack.provider.extensions.gardener.cloud/v1alpha1
+kind: WorkerConfig
+serverGroupConfig:
+  policy: soft-anti-affinity
+```
+
+Currently, the only configuration available is the `serverGroupConfig`. If this option is set, then a new server group will be created with the configured policy and all machines managed by this worker pool will be assigned as members of the group.
+
+Please note that the available options for `serverGroupConfig` are defined in the provider specific section of `CloudProfile`.
+
```

Comment:
Can you also explain whether it's possible to add/change/remove this for existing clusters/worker pools?